src/mono/wasm/templates/templates/console/Program.cs

```diff
@@ -1,3 +1,18 @@
 using System;
+using System.Runtime.InteropServices.JavaScript;
 
-Console.WriteLine("Hello, Console!");
+Console.WriteLine("Hello, Console");
```

Comment:
ah, there's one more issue. the test is now failing, because of that removed `!` char.
```
      Assert.Contains() Failure
      Not found: Hello, Console!
      In value:  [] WasmAppHost --runtime-config /datadisks/disk1/work/9EDF08E8/w/BE360A13/e/Release_0mfytptz.lb1/bin/Release/net7.0/browser-wasm/AppBundle//Release_0mfytptz.lb1.runtimeconfig.json
                 [] Running: node main.mjs
                 [] Using working directory: /datadisks/disk1/work/9EDF08E8/w/BE360A13/e/Release_0mfytptz.lb1/bin/Release/net7.0/browser-wasm/AppBundle
                 [] runArgs: {"applicationArguments":[],"runtimeArguments":[]}
                 [] mono_wasm_runtime_ready fe00e07a-5519-4dfe-b35a-f867dbaf2e28
                 [] Loaded 9239 symbols
                 [] Hello, World! Greetings from node version: v14.18.2
                 [] Hello, World! Greetings from node version: v14.18.2
                 [] Hello, Console
                 [-stderr] 
```
so either put it back or update the test to check for "Hello, Console" instead